### DISCUSSION TITLE
Choose a different interval on the top graph

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#282] [Anything Else]
Right now the top graph shows the interval according to the time period selected in the date picker. Here's an example:

![Screenshot from 2020-08-12 16-49-58](https://user-images.githubusercontent.com/34340819/90030071-e78d1780-dcbb-11ea-89e6-4ae6063dc9e8.png)

* If "Today" selected, the graph shows hourly intervals 
* If "Last 30 days" selected, it shows daily intervals
* If "Last 6 months" selected, it shows monthly intervals

Would be great with the option to manually select a different interval. For instance, I'd love to be able to select "weekly" interval for the graph when I'm looking on a longer term time period such as "last 3 months" or "last 6 months".

We could make this completely flexible and allow users to select whichever interval they want:

* If "Today" selected, the graph shows hourly intervals by default but I can switch it to "Daily" if I want
* If "Last 30 days" selected, it shows daily intervals by default but I can switch it to "Weekly" or "Monthly" if I want
* If "Last 6 months" selected, it shows monthly intervals by default but I can switch it to "Daily" or "Weekly" if I want

**UPDATE:** You can now set a different interval on the top graph to get a deeper level of detail and make it easier to spot trends.

**Comment by metmarkosaric** [comment]
Request for [this on Twitter](https://twitter.com/TommiUrtti/status/1366861659038314496): "it would be nice to get some options for granularity in this view. E.g. that I could look at 6 months data with 1 week intervals."

**Comment by reinder42** [comment]
Searched the forum for exactly this feature request – I'd love this! I'm looking often at the month-based view to spot trends, because their intervals are 'months' it's really hard to see week-to-week differences (interval → week). Same goes for Custom (days), which would benefit from a 'week' interval. I think each date range (today, last 30, mtd) could still have its (already sensible) default.

What would it take to get this on the roadmap?

**Reply by metmarkosaric** [reply]
thanks @reinder42! i added this idea myself and it's something i'd love to see too

about the roadmap: we have limited development resources so need to take it one step at a time with the focus being on most frequently requested items first. that's why things such as cities, user accounts and time on page have higher priority

**Reply by reinder42** [reply]
Of course, of course! And I love that gift icon that pops up every time you've built some new features :-)

**Comment by metmarkosaric** [comment]
**UPDATE:** You can now set a different interval on the top graph to get a deeper level of detail and make it easier to spot trends.
